react/features/chat/components/web/ChatInput.tsx

```diff
@@ -101,7 +93,9 @@ class ChatInput extends Component<Props, State> {
     componentDidMount() {
         if (isMobileBrowser()) {
             // Ensure textarea is not focused when opening chat on mobile browser.
-            this._textArea && this._textArea.blur();
+            if (this._textArea && this._textArea.current) {
```

Comment:
I think you could've just done `this._textArea?.current && this._textArea.current.blur()`